src/main/kotlin/string/addition/caculator/domain/Operand.kt

```diff
@@ -0,0 +1,20 @@
+package string.addition.caculator.domain
+
+@JvmInline
+value class Operand(val number: Int?) {
+
+    init {
+        number ?: run {
+            throw RuntimeException("An operand from the input string should be a Number")
+        }
```

Comment:
```suggestion
        number ?:  RuntimeException("An operand from the input string should be a Number")
```

이런 경우에는 스코프 함수를 사용할 필요가 없을 것 같네요 😄